src/mysql_query.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+#
+# igcollect - PostgreSQL query results
+#
+# This script executes a single query and prints the results.  The columns
+# returned by the query are going to be appended to the given prefix.
+# The query must return numeric values.
+#
+# Copyright (c) 2016, InnoGames GmbH
+#
+from __future__ import print_function
+
+import MySQLdb
+
+from argparse import ArgumentParser
+from time import time
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument('--prefix', default='mysql_query')
+    parser.add_argument('--dbname', default='mysql')
```

Comment:
`default='test'` might me better.  I don't think MySQL has a database called `mysql` by default.

Reply:
the default databases are:
mysql
performance_schema
information_schema
It is highly advised to remove the test database after install